crates/bevy_ecs/src/world/mod.rs

```diff
@@ -35,11 +36,45 @@ impl Default for WorldId {
     }
 }
 
-/// [World] stores and exposes operations on [entities](Entity), [components](Component),
+/// Stores and exposes operations on [entities](Entity), [components](Component),
 /// and their associated metadata.
 /// Each [Entity] has a set of components. Each component can have up to one instance of each
 /// component type. Entity components can be created, updated, removed, and queried using a given
 /// [World].
+///
+/// # Resources
+///
+/// The world can store *resources*, a special kind of components, not associated to any entity.
```

Comment:
I'm not sure I entirely agree with characterizing resources in this way.

The fact that they're currently component-like is basically just an implementation detail.